ingestion/src/metadata/utils/avro_parser.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Utils module to parse the avro schema
+"""
+
+import traceback
+
+import avro.schema as avroschema
+
+from metadata.utils.logger import ingestion_logger
+
+logger = ingestion_logger()
+
+
+def parse_avro_schema(schema):
```

Comment:
can you please add the signature here? Both schema type and return type. For these functions that might be tricky, as we have different packages going on, typing can help us understand what we are doing

Reply:
I'll add it 👍